README.md

```diff
@@ -1294,6 +1300,17 @@ relies on LibTorch/TorchScript (C++) which has been deprecated from
 So, users interested in new features introduced in PyTorch 2.0 should try the
```

Comment:
The current changes look good but I think we should remove the deprecation note from the PyTorch 2.0 section. I don't think there has been any announcements about deprecating torch script.

Reply:
My original note was this, rather than claiming any specific support/deprecation:
> Currently, the [PyTorch Backend](https://github.com/triton-inference-server/pytorch_backend)
doesn't support PyTorch 2.0, so users should try the Python Backend for these
models instead.

I do think we should point out PyTorch 2.0 in some way though.

What do you think?

Reply:
Also, I think we are using PyTorch 2.0 in PyTorch backend. It is just that `torch.compile` and other optimizations cannot be enabled in PyTorch backend.

Reply:
Exactly. Technically, we support pytorch 2.0 in our PyTorch backend. However, as the backend depends upon TorchScript which is no longer being maintained, users interested in new features introduced via pytorch 2.0 can not access them via pytorch backend.

Reply:
>  I don't think there has been any announcements about deprecating torch script.

If that's true we should just remove the deprecation note.

Reply:
Currently, the
[PyTorch Backend](https://github.com/triton-inference-server/pytorch_backend)
relies on LibTorch/TorchScript (C++). So, users interested in new workflows introduced in PyTorch 2.0 should try the ...

Reply:
I will remove the PyTorch 2.0 notes for now until we start seeing more usage. Related PR: https://github.com/triton-inference-server/pytorch_backend/pull/101